tutorials/get-started-with-dqlite-in-go.md

```diff
@@ -0,0 +1,23 @@
+This step-by-step guide walks you through the setup of an Dqlite example application. 
+By the end of this tutorial you will have a fully-functional Dqlite cluster deployed on your machine and you'll be able to interact with it through a HTTP client.
+
+## What you’ll need:
+
+* A workstation, e.g. a laptop
+* Some familiarity with Go
+
+## What you’ll do:
+
+* [Setup your development environment](tbd)
+* [Create an example application](tbd)
+
+## Next steps:
+
+In the next step we'll [setup your development environment](tbd).
```

Comment:
```suggestion
In the next step we'll [set up your development environment](tbd).
```
Just as an explanation: It's "to set up" as a verb, but "a setup" as a noun. So you will "set something up", and then you'll have a "setup".

Reply:
Thanks for the explanation - learned something today :)